yesod-auth/Yesod/Auth/Message.hs

```diff
@@ -60,6 +60,7 @@ data AuthMessage =
     | ProvideIdentifier
     | SendPasswordResetEmail
     | PasswordResetPrompt
+    | CurrentPassword
```

Comment:
It's the Yesod mailing list:

http://groups.google.com/group/yesodweb

You can find out about Yesod community resources at:

http://www.yesodweb.com/page/community